.github/renovate.json5

```diff
@@ -39,5 +39,9 @@
       "matchManagers": ["pre-commit"],
       "description": "Weekly update of pre-commit dependencies",
     },
-  ]
+  ],
+  "vulnerabilityAlerts": {
+    "commitMessageSuffix": "",
+    "labels": ["internal", "security"],
```

Comment:
This repo doesn't have `security` label. renovatebot couldn't add this label to its PRs.

Reply:
I'm pretty sure it would have just created one, but I've created one just to be sure: https://github.com/astral-sh/ruff/labels/security